go/signedexchange/cmd/gen-signedexchange/main.go

```diff
@@ -0,0 +1,111 @@
+package main
+
+import (
+	"encoding/pem"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"net/url"
+	"os"
+	"time"
+
+	"github.com/WICG/webpackage/go/signedexchange"
+)
+
+var (
+	flagUri            = flag.String("uri", "https://example.com/index.html", "The URI of the resource represented in the exchange")
+	flagResponseStatus = flag.Int("status", 200, "The status of the response represented in the exchange")
+	flagContent        = flag.String("content", "index.html", "Source file to be used as the exchange payload")
+	flagCertificate    = flag.String("certificate", "cert.pem", "Certificate chain PEM file of the origin")
+	flagCertificateUrl = flag.String("certUrl", "https://example.com/cert.msg", "The URL where the certificate chain is hosted at.")
+	flagValidityUrl    = flag.String("validityUrl", "https://example.com/resource.validity.msg", "The URL where resource validity info is hosted at.")
+	flagPrivateKey     = flag.String("privateKey", "cert-key.pem", "Private key PEM file of the origin")
+	flagOutput         = flag.String("o", "out.htxg", "Signed exchange output file")
+	flagMIRecordSize   = flag.Int("miRecordSize", 4096, "The record size of Merkle Integrity Content Encoding")
+)
+
+func run() error {
+	payload, err := ioutil.ReadFile(*flagContent)
+	if err != nil {
+		return fmt.Errorf("failed to read content from payload source file \"%s\". err: %v", *flagContent, err)
```

Comment:
`%q` will avoid the need for quotes around `"%s"`. https://golang.org/pkg/fmt/#hdr-Printing

Reply:
Done.